test-basic/ssl-min-allow-tls-test.js

```diff
@@ -12,7 +12,7 @@ let serverConfiguration = {};
 let host  = testconfig.testHost;
 
 describe('document write and read using min tls', function () {
-    this.timeout(10000);
+    this.timeout(12000);
```

Comment:
Was this failing for you locally too? It's another test that I don't think has failed on Jenkins in quite a while. Of course, 10s is already totally arbitrary, so changing it to 12s doesn't really hurt.

Reply:
Yes the timing for the error ssl v1.2 error test was like 11000ms for me. so 120000 timeout fixed locally. But totally environmental and not even close to a good solution. I could bump to 200000.